docs/source/man.rst

```diff
@@ -308,84 +308,145 @@ As the network infrastructure changes, it might be necessary to delete some sour
+Use the ``qpc scan`` command to create, run and manage scans.
 
-**IMPORTANT:** If any ssh-agent connection is set up for a target host, that connection will be used as a fallback connection.
+A scan defines a collection of network information, including the sources to scan and additional options, such as elective products to omit from the scan and the maximum number of parallel system scans. The creation of a scan references information such as sources so that the act of running the scan is repeatable, without a requirement to reenter network information for each scan attempt.
+
+Creating and Editing Scans
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Use the ``qpc scan add`` command to create scan objects with one or more sources. This command creates a scan object that references the supplied sources and contains any options supplied by the user. Each instance of a scan object is assigned a unique *identifier* to identify the scan jobs that are associated with that scan.
 
-**qpc scan start --sources=** *source_list* **[--max-concurrency=** *concurrency* **]** **--disable-optional-products=** *products_list*
+**qpc scan add --name** *name* --sources=** *source_list* **[--max-concurrency=** *concurrency* **]** **--disable-optional-products=** *products_list*
 
 ``--sources=source_list``
 
   Required. Contains the list of source names to use to run the scan.
 
 ``--max-concurrency=concurrency``
 
-  Contains the maximum number of parallel system scans. If this value is not provided, the default is ``50``.
+  Optional. Contains the maximum number of parallel system scans. If this value is not provided, the default is ``50``.
 
 ``--disable-optional-products=products_list``
 
-  The product inspection exclusion. Contains the list of products to exclude from inspection. Valid values are jboss_eap, jboss_fuse, and jboss_brms.
+  Optional. The product inspection exclusion. Contains the list of products to exclude from inspection. Valid values are jboss_eap, jboss_fuse, and jboss_brms.
+
+The information in a scan might change as the structure of the network changes. Use the ``qpc scan edit`` command to edit an existing scan to accommodate those changes.
+
+Although ``qpc scan`` options can accept more than one value, the ``qpc scan edit`` command is not additive. To edit a scan and add a new value for an option, you must enter both the current and the new values for that option. Include only the options that you want to change in the ``qpc scan edit`` command. Options that are not included are not changed.
+
+**qpc scan edit --name** *name* --name** *name* --sources=** *source_list* **[--max-concurrency=** *concurrency* **]** **--disable-optional-products=** *products_list*
+
+For example, if a scan contains a value of ``network1sources`` for the ``--sources`` option, and you want to change that scan to use both the ``network1sources`` and ``network2sources`` sources, you would edit the scan as follows:
```

Comment:
would be better with singular names:
`network1source` and `network2source`

Reply:
or better `satellite1source` to show you can mix source types.

Reply:
Done